src/runtime_src/core/tools/xbutil2/OO_HostMem.cpp

```diff
@@ -93,6 +95,7 @@ OO_HostMem::execute(const SubCmdOptions& _options) const
     throw xrt_core::error(std::errc::operation_canceled);
   }
   if(m_devices.empty()) {
+    std::cerr << boost::format("ERROR: A device needs to be specified.\n");
     printHelp();
```

Comment:
I don't think we print out help when a device is not specified. For consistency purposes, please remove this. See p2p:
```
xbutil configure --p2p enable
ERROR: A device needs to be specified.
```
For the future, if a device is not specified, we should show the error message and optionally show the list of available devices, for eg:
```
$ xbutil validate 

ERROR: Please specify a single device using --device option

List of available devices:
  [0000:b3:00.1] : xilinx_u200_gen3x16_xdma_base_1
  [0000:18:00.1] : xilinx_u30_gen3x4_base_2
  [0000:17:00.1] : xilinx_u30_gen3x4_base_2

```

Reply:
Fixed!